package.json

```diff
@@ -37,10 +33,8 @@
   },
   "homepage": "https://github.com/Geode-solutions/GeodeApp#readme",
   "devDependencies": {
-    "@electron-forge/cli": "^7.4.0",
-    "@electron-forge/maker-zip": "^7.4.0",
-    "@electron-forge/publisher-github": "^7.4.0",
     "electron": "^28.2.7",
+    "electron-builder": "^24.13.3",
     "nuxt": "^3.12.3",
     "nuxt-electron": "^0.7.0"
   },
```

Comment:
maybe try to clean as much dependencies as possible to devdependencies to reduce package size

Reply:
with a nuxt genreate, not sure if you still need some...